<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addJornal</class>
 <widget class="QWidget" name="addJornal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Título:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>140</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>180</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mês:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>220</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ano:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>260</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editora:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addjornal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Incluir jornal(is)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoTITULO">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>100</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoDIA">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>140</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoMES">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>180</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoANO">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>220</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoEDITORA">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>260</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delJornal">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Excluir jornal(is)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>300</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quantidade:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoQUANTIDADE">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>300</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Voltar ao Menu</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>commandLinkButton</tabstop>
  <tabstop>campoTITULO</tabstop>
  <tabstop>campoDIA</tabstop>
  <tabstop>campoMES</tabstop>
  <tabstop>campoANO</tabstop>
  <tabstop>campoEDITORA</tabstop>
  <tabstop>campoQUANTIDADE</tabstop>
  <tabstop>btn_addjornal</tabstop>
  <tabstop>btn_delJornal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
